pyensign/exceptions.py

```diff
@@ -27,6 +117,14 @@ class EnsignError(PyEnsignError):
     pass
 
 
+class EnsignRPCError(EnsignError):
```

Comment:
It would probably be wise to create a story to parse the RPC error for gRPC status code and message so that we could distinguish different types of errors from Ensign a bit more easily.